### PR TITLE
feat(cxl-ui): dashboard roadmap block

### PR DESCRIPTION
**Pull request by HenerHoop**
https://app.clickup.com/t/86aydn6th

### CODE REVIEW

**Comment by HenerHoop** [comment]
Task linked: [CU-86aydn6th Roadmap Block](https://app.clickup.com/t/86aydn6th)

**Review by pawelkmpt — COMMENTED** [review]
@HenerHoop I'm adding visual glitches here, but I'm not sure if they need adjustment in aybolit or WPS.

**1. Too big whitespace** 
Figma seems to have it smaller. It just feels empty anyway.

Screenshot 2023-11-02 at 10 10 22

---

**2. Shadow behind the content** 

Screenshot 2023-11-02 at 10 11 04

It probably depends on some other branch. If so, please point me out  to it.

---

**3. Menu looks different**

This looks like a side effect of changes. I'll make sure I compiled everything correctly but wanted to highlight it anyway, maybe you see the same.

Screenshot 2023-11-02 at 10 11 22

**Review by pawelkmpt — CHANGES_REQUESTED** [review]
Few visual things. Check comments above.

**Comment by HenerHoop** [comment]
> @HenerHoop I'm adding visual glitches here, but I'm not sure if they need adjustment in aybolit or WPS.
> 
> **1. Too big whitespace** Figma seems to have it smaller. It just feels empty anyway.
> 
> Screenshot 2023-11-02 at 10 10 22
> 
> **2. Shadow behind the content**
> 
> Screenshot 2023-11-02 at 10 11 04
> 
> It probably depends on some other branch. If so, please point me out to it.
> 
> **3. Menu looks different**
> 
> This looks like a side effect of changes. I'll make sure I compiled everything correctly but wanted to highlight it anyway, maybe you see the same.
> 
> Screenshot 2023-11-02 at 10 11 22

@pawelkmpt 

1. Fixed
2. Removed
3. I tried in both repos while being in the master branch, and it displays the same way.

File: packages/cxl-ui/src/components/cxl-light-card.js [hunk 1]
```diff
@@ -14,9 +14,9 @@ export class CXLLightCardElement extends CXLBaseCardElement {
     return [...super.styles, cxlLightCardStyles];
   }
 
-  @property({ type: Number }) progress = 0;
+  @property({ type: String, attribute: 'progress-completed' }) progressCompleted = '0';
 
-  @property({ type: Number }) lessons = 0;
+  @property({ type: String, attribute: 'progress-total' }) progressTotal = '0';
```
Comment (pawelkmpt) [inline packages/cxl-ui/src/components/cxl-light-card.js:19]: Why are these better as string, not number?

**Review by pawelkmpt — CHANGES_REQUESTED** [review]
_Edit roadmap_ button needs fixing on mobile and tablet screens.

Screenshot 2023-11-02 at 20 27 06
Screenshot 2023-11-02 at 20 28 26

File: packages/cxl-ui/scss/cxl-dashboard-header.scss [hunk 2]
```diff
@@ -33,191 +59,98 @@
...
-}
 
-::slotted(.stats-mobile) {
-  @media #{mq.$large} {
-    display: none;
+  .content {
+    & + .content {
+      margin-top: var(--lumo-space-l);
+      padding-top: var(--lumo-space-l);
+      border-top: 1px solid #d2d5da;
```
Comment (pawelkmpt) [inline packages/cxl-ui/scss/cxl-dashboard-header.scss:153]: Don't we have this color in variable? Or something similar?

File: packages/storybook/cxl-ui/cxl-dashboard-header/template.js [hunk 3]
```diff
@@ -12,74 +14,38 @@ export const DashboardHeaderTemplate = (header) => html`
...
-    .cta3Link=${header.cta3Link}
-    ?hasRoadmap=${header.hasRoadmap}
+    .showCompletedStats=${header.showCompletedStats}
+    .showContinueSlider=${header.showContinueSlider}
+    .showRoadmap=${header.showRoadmap}
+    .showRoadmapStats=${header.showRoadmapStats}
+    .showRoadmapSlider=${header.showRoadmapSlider}
+    .editRoadmapLinkUrl=${header.editRoadmapLinkUrl}
+    .createRoadmapLinkUrl=${header.createRoadmapLinkUrl}
+    .showMinidegrees=${header.showMinidegrees}
```
Comment (pawelkmpt) [inline packages/storybook/cxl-ui/cxl-dashboard-header/template.js:24]: Are these suppose to be `camelCase` or `kebab-case` too?

**Review by pawelkmpt — COMMENTED** [review]
@HenerHoop one thing caught my eye. Please take a look

**Pull request merged**